debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/ObjectFieldInfo.java

```diff
@@ -60,6 +70,11 @@
 	int superclassBackfillOffset; /* inherited backfill */
 	int myBackfillOffset; /* backfill available for this class's fields */
 	int subclassBackfillOffset; /* backfill available to subclasses */
+	boolean isValue = false;
+	J9FlattenedClassCachePointer flattenedClassCache = J9FlattenedClassCachePointer.NULL;
```

Comment:
Use of the type `J9FlattenedClassCache` will be problematic. Older core files, or those produced by Java8 VMs, won't have that type: expect exceptions like `NoClassDefFoundError`.